neuralop/losses/fourier_diff.py

```diff
@@ -0,0 +1,72 @@
+from neuralop.layers.fourier_continuation import FCLegendre, FCGram
+import torch
+import warnings
+
+def fourier_derivative_1d(u, order=1, L=2*torch.pi, use_FC=False, FC_d=4, FC_n_additional_pts=40, FC_one_sided=False, low_pass_filter_ratio=None):
+    """
+    Compute the 1D Fourier derivative of a given tensor.
+    Use Fourier continuation to extend the signal if it is non-periodic. 
+    Use with care, as Fourier continuation and Fourier derivatives are not always stable.
```

Comment:
This is helpful info, but would be much more helpful if you could provide more context on *when* and *where* FC derivatives are stable, or what kind of signs people might look for. It might even be cool to demonstrate in an example, if there are simple principles to follow that can be easily illustrated.